hack/ovn-kind-cno.sh

```diff
@@ -175,6 +175,20 @@ data:
       replicas: ${NUM_MASTER_NODES}
 EOF
 
+# To test the hack that will allow an installation to change the gateway mode
```

Comment:
nit: Not sure if we wan't to add this here, maybe just add a quick note to the KIND README?

Reply:
done. see if you like how it looks now. just a few lines in the readme and points to the hack/gateway-mode.yaml file as
the example to use.